### PR TITLE
Fix freezing of configurator while loading local firmware and OSD preview while moving elements on Linux

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: 
#2911 
#2878

- [x] Update workflow to use Ubuntu 22.04 LTS (needed to make this PR build)
- [x] Update Node to 16.x. superseeds https://github.com/betaflight/betaflight-configurator/pull/2782
- [x] Update NW.js to 0.62 (synchronized versions on all platforms)
- [x] Remove OSD dragging preview on linux (as this was an issue after 0.54.1)

Reminder: Update ci.yml as Ubuntu 22.04 is released.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: .github/workflows/ci.yml
Comment (McGiverGim) [inline .github/workflows/ci.yml:19]: We decided in the latest PR about this to use the version here, to be sure in what version we build. In this way it will not fail when changing from one version to another without noticing.

Comment (haslinghuis) [reply .github/workflows/ci.yml:19]: Got a `waiting for a runner to pickup this job` error using `ubuntu-22.04`. Might still work with ubuntu-20.04 in ci.yml 

Strange enough using `ubuntu-22.04` does work on the other workflows.

Comment (McGiverGim) [reply .github/workflows/ci.yml:19]: According to this https://github.com/actions/virtual-environments/discussions/5427 we will need to wait some more days for Ubuntu 22.04.

Comment (haslinghuis) [reply .github/workflows/ci.yml]: For now 20.04 works. Will update ci.yml later.

**Review by limonspb — DISMISSED** [review]

**Review by limonspb — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Comment by haslinghuis** [comment]
@McGiverGim updated the PR. 

Seems there is still work ongoing but all seems to work now,

image

File: src/js/tabs/osd.js
Comment (McGiverGim) [inline src/js/tabs/osd.js:2358]: I have not seen how this looks, but I suppose that if you add it, is because is better to not show anything than show a rectangle.

**Review by McGiverGim — APPROVED** [review]
I suppose this has been tested in our repo. It seems that until merged, here in Betaflight continues using 20.04. But the code seems correct to me.

**Pull request merged**
